elixir/apps/web/assets/js/hooks.js

```diff
@@ -14,6 +14,40 @@ Hooks.Tabs = {
   },
 };
 
+Hooks.Analytics = {
+  mounted() {
+    this.handleEvent("identify", ({ id, account_id, name, email }) => {
+      var mixpanel = window.mixpanel || null;
+      if (mixpanel) {
+        mixpanel.identify(id);
+        mixpanel.people.set({ $email: email, $name: name });
```

Comment:
Hmmm can we easily scrub this email and name data from Mixpanel later on? We need to be able to service DSAR requests (or maybe it's not worth tracking it?)

Reply:
@jamilbk we can delete the profile in Mixpanel. I kept it only so that we can cross-reference customer journeys, we sync it it in HubSpot anyway

Reply:
K, I think it's fine. I just need to go update some legal docs.